backend/routes/XrplNFTHelper.js

```diff
@@ -0,0 +1,462 @@
+          - Account: this.transactionDetails.Account,
+          - NFTokenID: this.transactionDetails.NFTokenID
+
+  Returns: Transaction Result string.
+*
+*/
+  async burnNFT(){
+    try {
+     
+
+      const wallet = xrpl.Wallet.fromSeed(this.transactionDetails.Secret)
+      const client = new xrpl.Client(this.clientDetails)
+      await client.connect()
+
+      console.log("Connected to Sandbox..burning single NFT.")
+
+   
+    
+    const transactionData = {
+          TransactionType: this.transactionDetails.TransactionType,
+          Account: this.transactionDetails.Account,
+          NFTokenID: this.transactionDetails.NFTokenID
+      }
+      
+    
+      
+      const tx = await client.submitAndWait(transactionData,{wallet})
+  
+      client.disconnect()
+      return tx.result.meta.TransactionResult
+
+}
+catch(err) {
+  console.log("Error occured during minToken() call" + err)
+  return;
+}
+  
+  }
+
+
+  /*Burn all NFTs in the account
+
+  Params (required): 
+  - transactionDetails.Secret
+  - transactionDetails.Account
+
+  Returns: 
+  Array of NFTokenID's for removal of metadata storage
+
+
+  */
+
+
+  async burnAllNFT(){
+    try {
+     
+      const wallet = xrpl.Wallet.fromSeed(this.transactionDetails.Secret)
+      const client = new xrpl.Client(this.clientDetails)
+      await client.connect()
+
+      console.log("Connected to Sandbox..burning ALL NFT's for specified account.")
+
+      let nfts = await client.request({
+        method: "account_nfts",
+        account: this.transactionDetails.Account
+        })
+
+        console.log("Attempting to burn " + nfts.result.account_nfts.length + " NFT's..")
+              
+        for (let index = 0; index < nfts.result.account_nfts.length; index++) {
+
+
+          const transactionData = {
+            TransactionType: this.transactionDetails.TransactionType,
+            Account: this.transactionDetails.Account,
+            NFTokenID: nfts.result.account_nfts[index].NFTokenID
+        }
+
+
+          const tx = await client.submitAndWait(transactionData,{wallet})
+          console.log("Burnt " + nfts.result.account_nfts[index].NFTokenID + " ")
+      
+        }
+        
+        console.log("END.. All NFT's burned")
+        return nfts.result.account_nfts;
+
+}
+catch(err) {
+  console.log("Error occured during burnAllNFT() call" + err)
+  return;
+}
+  
+  }
+
+  async acctInfo(){
+
+
+    const client = new xrpl.Client(this.clientDetails) 
+
+    await client.connect()
+
+    const account_info = await client.request({
+      "command": "account_info",
+      "account": this.transactionDetails.Account
+    })
+
+    client.disconnect()
+
+    return account_info
+  }
+
+  async getTicketInfo(){
+
+
+    const client = new xrpl.Client(this.clientDetails) 
+
+    await client.connect()
+
+    let response = await client.request({
+      "command": "account_objects",
+      "account": this.transactionDetails.Account,
+      "type": "ticket"
+    })

```

Comment:
This will run into the pagination problem as well - might be worth making a helper which can do the pagination part for you.